minor tweaks to servicecontrol/servicecontrol-instances/hardware.md (#6084)
Particular/docs.particular.net: servicecontrol/servicecontrol-instances/hardware.md

### Ongoing server performance monitoring

The requirements for a server hosting ServiceControl may change over time as the system evolves. It's important to continuously monitor the CPU, RAM, disk I/O, and network I/O for the server running ServiceControl to ensure adequate resources are available for overall system health.

Disk, CPU, RAM, and network performance may be monitored using the Windows Resource Monitor and/or Windows Performance counters.

### Storage recommendations

* Store ServiceControl data on a dedicated disk. This makes low-level resource monitoring easier and ensures applications are not competing for storage IOPS.
* Store multiple ServiceControl databases on separate physical disks to prevent multiple instances competing for the same disk resources.
* Disable disk write caching (read caching can remain enabled) to prevent data corruption if the (virtual) server or disk controller fails. This is a general best practice for databases.

> * [Database paths](/servicecontrol/creating-config-file.md#host-settings-servicecontroldbpath) should be located on disks suitable for low latency write operations (e.g. fiber, solid state drives, raid 10), with a recommended IOPS of at least 7500.

Author: Futher up we refer to "ServiceControl data" and "ServiceControl databases", yet here (and on line 57), we switch to "Database paths". Should we be constistent?

Reviewer: I actually didn't know enough to know they were referring to the same thing. If they are, then yes, definitely, we should be consistent!

Reviewer 2: As I originally wrote this, with data I meant everything in the DB Path which is not only databases. Maybe we can use CONFIGURED DATABASE PATH, but we are not consistent in SCMU either. For me servicecontrol STORAGE path is a better generic name that would include databases, indexes, logs and failed messages.

Author: Let's leave it for now. As discussed, these docs are up for review soon.

### Message size / MaxBodySizeToStore

> In general, the smaller the messages, the faster ServiceControl will process audit records (see [this blog post](https://particular.net/blog/putting-your-events-on-a-diet)). For larger message payloads, consider using the [data bus feature](/nservicebus/messaging/databus/).

Reviewer 2: Please revert the THIS reference
```suggestion
In general, the smaller the messages, the faster ServiceControl will process audit records (guidance on reducing message size is available in the blog post [putting your events on a diet](https://particular.net/blog/putting-your-events-on-a-diet)). For larger message payloads, consider using the [data bus feature](/nservicebus/messaging/databus/).
```

Author: why?

Author: OK, how about:

```suggestion
In general, the smaller the messages, the faster ServiceControl will process audit records (see [_Putting your events on a diet_](https://particular.net/blog/putting-your-events-on-a-diet)). For larger message payloads, consider using the [data bus feature](/nservicebus/messaging/databus/).
```

> In general, the smaller the messages, the faster ServiceControl will process audit records (see [_Putting your events on a diet_](https://particular.net/blog/putting-your-events-on-a-diet)). For larger message payloads, consider using the [data bus feature](/nservicebus/messaging/databus/).

Author: @ramonsmits CI fails because of the word "your" in the blog post title. Maybe this instead?

```suggestion
In general, [the smaller the messages](https://particular.net/blog/putting-your-events-on-a-diet), the faster ServiceControl will process audit records. For larger message payloads, consider using the [data bus feature](/nservicebus/messaging/databus/).
```

For audit messages, lower the [`ServiceControl/MaxBodySizeToStore`](/servicecontrol/creating-config-file.md#performance-tuning-servicecontrolmaxbodysizetostore) setting to skip storage of larger audit messages. This setting will only reduce load if non-binary [serialization](/nservicebus/serialization/) is used.

WARNING: When using ServiceInsight, the message body is not viewable for messages that exceed the `ServiceControl/MaxBodySizeToStore` limit.

### Separate disks for database and index files

> Besides using a dedicated disk for the ServiceControl [database paths](/servicecontrol/creating-config-file.md#host-settings-servicecontroldbpath), it's possible to store the embedded database index files on a separate disk. Use the [`Raven/IndexStoragePath`](/servicecontrol/creating-config-file.md#host-settings-ravenindexstoragepath) setting to change the index storage location.

Reviewer 2: I'm not sure if this actually still works with Raven 5.

Author: As discussed, these docs are due for review soon so we'll leave this for then.
